.github/workflows/java-gradle-docker.yaml

```diff
@@ -97,12 +109,14 @@ jobs:
 
     steps:
       - name: Build tarball image
-        uses: bakdata/ci-templates/actions/java-gradle-build-jib@v1.16.0
+        uses: bakdata/ci-templates/actions/java-gradle-build-jib@feature/improved-gradle-caching
         with:
           java-distribution: ${{ inputs.java-distribution }}
           java-version: ${{ inputs.java-version }}
           gradle-version: ${{ inputs.gradle-version }}
           gradle-cache: ${{ inputs.gradle-cache }}
+          gradle-cache-read-only: true
```

Comment:
I get that we don't want to write the cache for the test step but why not here and in publish?

Reply:
For publish I was unsure if we want to write the cache because I don't fully trust the Nexus plugin. For Jib build we could do it I guess. What do you think?

Reply:
Probably it is safer to just disable it, the layers are cached by the docker registry, so I don't think we would gain a lot. But I think we should then include that `gradle-cache-read-only`  is only used for the build + test steps in the description

Reply:
Added it to the docs